### PR TITLE
Updated broken "tree shaking guide" in the readme file.

### LINKED ISSUE #6295: [docs] Tree Shaking Guide not available
#### Expected behavior

Wanted to read about tree shaking when I found the link on github
image

#### Actual behavior
https://docs.web3js.org/docs/guides/web3_tree_shaking_support_guide

Page is not available

### PR DESCRIPTION
**Pull request by tiseysoft**
## Description

Updated web3.js tree shaking guide broken link

Fixes #6295

## Type of change

- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x ] I have selected the correct base branch.
- [x ] I have performed a self-review of my own code.
- [x ] I have commented my code, particularly in hard-to-understand areas.
- [x ] I have made corresponding changes to the documentation.
- [x ] My changes generate no new warnings.
- [x ] Any dependent changes have been merged and published in downstream modules.
- [x ] I ran `npm run lint` with success and extended the tests and types if necessary.
- [x ] I ran `npm run test:unit` with success.
- [x ] I ran `npm run test:coverage` and my test cases cover all the lines and branches of the added code.
- [x ] I ran `npm run build` and tested `dist/web3.min.js` in a browser.
- [x ] I have tested my code on the live network.
- [x ] I have checked the Deploy Preview and it looks correct.
- [x ] I have updated the `CHANGELOG.md` file in the root folder.
- [x ] I have linked Issue(s) with this PR in "Linked Issues" menu.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1815,6 +1815,10 @@ If there are any bugs, improvements, optimizations or any new feature proposal f
 
 -   Fixed: "'disconnect' in Eip1193 provider must emit ProviderRpcError #6003".(#6230)
 
+#### broken link 
+
+- Fixed: broken link in the readme file of the root folder.
+
```
Comment (jdevcs) [inline CHANGELOG.md]: @tiseysoft this change is not required to mention in changelog,

**Comment by jdevcs** [comment]
@tiseysoft Thanks for your contribution.

**Comment by tiseysoft** [comment]
> @tiseysoft Thanks for your contribution.

Updated, thank you.

**Review by Muhammad-Altabba — APPROVED** [review]
Thanks

**Review by jdevcs — APPROVED** [review]

**Pull request merged**